src/Prometheus/Storage/Redis.php

```diff
@@ -22,7 +22,7 @@ class Redis implements Adapter
         'host' => '127.0.0.1',
         'port' => 6379,
         'timeout' => 0.1,
-        'read_timeout' => 10,
+        'read_timeout' => "10",
```

Comment:
The standard is to use `'` instead of `"` for a string. What do you think of updating it?

Reply:
Would definately use single quotes for this

Reply:
Sure! I have created a new commit with single quotes rather than double ones.